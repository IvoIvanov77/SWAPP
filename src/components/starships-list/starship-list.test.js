import React from 'react';
import { shallow } from 'enzyme';
import StarshipList from './starships-list.component';

it('should render StarshipList component', () => {
  //   expect(shallow(<StarshipList />)).toMatchSnapshot();
});

describe('ChatactersList component', () => {
  let wrapper;
  const mockedStarships = [{ id: 1 }, { id: 2 }];

  beforeEach(() => {
    wrapper = shallow(<StarshipList starships={mockedStarships} />);
  });

  it('should render StarshipList component', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render starships', () => {
    const starships = wrapper.find('div').children();
    expect(starships.length).toBe(mockedStarships.length);
  });
});
