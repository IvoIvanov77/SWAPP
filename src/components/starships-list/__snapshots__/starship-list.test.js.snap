// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatactersList component should render StarshipList component 1`] = `ShallowWrapper {}`;
